<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sigma - Import Preparation Times</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../assets/Sigma.ico</normaloff>../assets/Sigma.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="bgWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#bgWidget{
background-color:#F8F7FC;}</string>
    </property>
    <widget class="QPushButton" name="continueButton">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>490</y>
       <width>150</width>
       <height>42</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;
font: 14pt &quot;Montserrat SemiBold&quot;;
color:white;
  background-color: #6F51EC;</string>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QPushButton" name="importButton">
     <property name="geometry">
      <rect>
       <x>606</x>
       <y>140</y>
       <width>114</width>
       <height>32</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Montserrat;
font-style: normal;
font-weight: 600;
font-size: 14px;
color:#6F51ED;
border: 1px solid #6F51ED;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Import</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="prepTabs">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>210</y>
       <width>601</width>
       <height>251</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QPushButton" name="backButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>77</y>
       <width>95</width>
       <height>32</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;
border: 1px solid #6F51ED;
font: 18pt &quot;Montserrat&quot;;
font-weight: 600;
font-size: 12px;
color:#6F51ED;</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>133</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#0B0B0B;

font-family: Montserrat;
font-style: normal;
font-weight: bold;
font-size: 24px;</string>
     </property>
     <property name="text">
      <string>Data Input</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>167</y>
       <width>401</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Montserrat;
font-style: normal;
font-weight: normal;
font-size: 12px;
line-height: 15px;
color:#ABB1C1;</string>
     </property>
     <property name="text">
      <string>Preparation Times for each machine</string>
     </property>
    </widget>
    <widget class="QWidget" name="sidebar" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>80</width>
       <height>600</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#373B54;</string>
     </property>
     <widget class="QPushButton" name="logo">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../assets/Sigma.png</normaloff>../../assets/Sigma.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="error">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>371</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: 'Montserrat Medium';
font-style: normal;
font-weight: normal;
font-size: 12px;
line-height: 15px;
color: rgb(255, 49, 52)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
